source/draw/gpu/opengl/shaders/unpackRgbaToMonochrome.frag

```diff
@@ -0,0 +1,10 @@
+#version 300 es
+precision mediump float;
+uniform sampler2D texture0;
+uniform int targetWidth;
+in highp vec4 fragmentTextureCoordinate;
+out float outColor;
+void main() {
+  int pixelIndex = int(float(targetWidth) * fragmentTextureCoordinate.z) % 4;
+  outColor = texture(texture0, fragmentTextureCoordinate.xy)[pixelIndex];
```

Comment:
By `soft tabs` you mean spaces? Because it allows people to set their own tab width. It can be discussed, but not here, please...